src/test/java/org/htmlunit/html/HtmlImageTest.java

```diff
@@ -318,4 +318,39 @@ public void clickWithCoordinates() throws Exception {
         img.click(2, 7);
         assertEquals(getExpectedAlerts()[0] + getExpectedAlerts()[1], page.getTitleText());
     }
+
+    /**
+     * @throws Exception on test failure
+     */
+    @Test
+    @Alerts({"in-out-Image.onload-", "onmousedown-in-out-Image.onload-onmouseup-in-out-onclick-in-out-Image.onload-Image.onload-"})
```

Comment:
you could shorten it slightly like this.. 🤷 
```suggestion
    @Alerts({"in-out-Image.onload-", "mousedown-in-out-Image.onload-mouseup-in-out-click-in-out-Image.onload-Image.onload-"})
```